config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-manual-job-config.yaml

```diff
@@ -32,12 +32,17 @@ presubmits:
           value: "1.21.0"
         - name: CSI_PROW_DEPLOYMENT
           value: "kubernetes-distributed"
-        - name: CSI_PROW_DRIVER_VERSION
-          value: "v1.7.2"
         - name: CSI_SNAPSHOTTER_VERSION
           value: "v3.0.3"
         - name: CSI_PROW_TESTS
           value: "sanity serial parallel serial-alpha parallel-alpha"
+        - name: CSI_PROW_SANITY_POD
+          # The hostpath pod is part of a DaemonSet with an unpredictable
+          # name. There's also more than one. We can list all them by
+          # their label, but then still need to filter by node name
+          # to find the one that is on the same host as the socat pod
+          # which exposes the CSI socket for us.
+          value: kubectl get pods -l app.kubernetes.io/name=csi-hostpathplugin -o "jsonpath={.items[?(@.spec.nodeName==\"$(kubectl get pods/csi-hostpath-socat-0 -o jsonpath={.spec.nodeName})\")].metadata.name}"
```

Comment:
I hope that YAML preserves quotation marks here. The shell command itself works when embedded as-is in an env variable.